mlir/include/mlir/Dialect/Affine/IR/AffineOps.td

```diff
@@ -63,10 +64,6 @@ def AffineApplyOp : Affine_Op<"apply", [Pure]> {
   // has a constant builder. That way we wouldn't need to explicitly specify the
   // result types here.
   let builders = [
-    OpBuilder<(ins "AffineMap":$map, "ValueRange":$mapOperands),
-    [{
-      build($_builder, $_state, $_builder.getIndexType(), map, mapOperands);
-    }]>,
```

Comment:
I don't quite understand the relationship between `InferTypeOpInterface` and these being autogenerated but that's what's happening here :shrug:.